rust-semantics/preprocessing/constants.md

```diff
@@ -24,5 +24,10 @@ module RUST-CONSTANTS
             </constant>
             ...
         </constants>
+
```

Comment:
This rule should actually be in a new file, called `rust-semantics/expression/constants.mx`. This file (and its module) should be required/imported in the same places as `rust-semantics/expression/literals.md`, so that it will be used both for preprocessing and for execution.